src/cascadia/TerminalSettingsModel/ActionArgs.idl

```diff
@@ -146,6 +146,7 @@ namespace Microsoft.Terminal.Settings.Model
 
     [default_interface] runtimeclass CopyTextArgs : IActionArgs
     {
+        CopyTextArgs();

```

Comment:
I missed where we would need to construct one of these

Reply:
Ah, this was needed in an elder version of `TerminalPage::_PopulateContextMenu`. But now we don't need that